Header header
float64[] coef_x
float64[] coef_y
float64[] coef_z
float64[] time
uint32  order